### PR TITLE
enforce namespace presence

### LINKED ISSUE #127: Enforce namespace (e.g., no.met) requirement

### PR DESCRIPTION
**Pull request by charlienegri**
**Summary**: closes #127

**Reviewer checklist**:

* [ ] The headers of all files contain a reference to the repository license
* [ ] 100% test coverage of new code - meaning:
  * [ ] The overall test coverage increased or remained the same as before
  * [ ] Every function is accompanied with a test suite
  * [ ] Tests are both positive (testing that the function work as intended with valid data) and negative (testing that the function behaves as expected with invalid data, e.g., that correct exceptions are thrown)
  * [ ] Functions with optional arguments have separate tests for all options
* [ ] Examples are supported by doctests
* [ ] All tests are passing
* [ ] All names (e.g., files, classes, functions, variables) are explicit
* [ ] Documentation (as docstrings) is complete and understandable

The checklist is based on the S-ENDA conventions and definition of done (see [General Conventions](https://s-enda-documentation.readthedocs.io/en/latest/general_conventions.html)). The above points are not necessarily relevant to all contributions. In that case, please add a short explanation to help the reviewer.

### CODE REVIEW

File: dmci/api/worker.py [hunk 1]
```diff
@@ -176,29 +176,23 @@ def _extract_metadata_id(self, xml_doc):
...
+                if len(words) != 2:
                     logger.warning("metadata_identifier not formed as namespace:UUID")
                     return False
+                namespace, fileUUID = words
 
                 logger.info("XML file metadata_identifier namespace:%s" % namespace)
                 logger.info("XML file metadata_identifier UUID: %s" % fileUUID)
                 break
 
-        if fileUUID is None:
```
Comment (johtoblan) [inline dmci/api/worker.py:198]: this cant be removed, since we can iterate through a file a and not find an UUID

Comment (charlienegri) [reply dmci/api/worker.py:198]: right, will revert

Comment (charlienegri) [reply dmci/api/worker.py:198]:
```
         if fileUUID is None:
             logger.warning("No UUID found in XML file")
             return False
         if namespace is None:
             logger.warning("No namespace found in XML file")
             return False
```

**Comment by charlienegri** [comment]
I fixed the tests

**Review by johtoblan — APPROVED** [review]
LGTM

**Pull request merged**